### PR TITLE
Expand test-index.js

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

File: test/test-index.js [hunk 1]
```diff
@@ -142,6 +130,17 @@ describe('index.js', function() {
     });
   });
 
+  it('should respect filter urls', function(done) {
+    var url = 'rootUrl';
```
Comment (kjin) [inline test/test-index.js]: Can you also add a URL that the policy passes?

Comment (matthewloring) [inline test/test-index.js]: done

File: test/test-index.js [hunk 2]
```diff
@@ -106,29 +106,17 @@ describe('index.js', function() {
 
   it('should not allow nested root spans', function(done) {
     agent.runInRootSpan({name: 'root', url: 'root'}, function(rootSpan1) {
-      var finished = false;
-      var finish = function () {
-        assert(!finished);
-        finished = true;
+      setTimeout(function() {
+        agent.runInRootSpan({name: 'root2', url: 'root2'}, function(rootSpan2) {
+          assert.ok(!rootSpan2);
```
Comment (ofrobots) [inline test/test-index.js]: nit: more readable/accurate to write `assert.stricEqual(rootSpan2, null)`

Comment (matthewloring) [inline test/test-index.js]: Done.

**Comment by matthewloring** [comment]
PR-URL: https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/409

**Pull request merged**